dockerfile-mode.el

```diff
@@ -42,6 +42,12 @@
   :type 'hook
   :group 'dockerfile)
 
+(defcustom dockerfile-mode-use-podman nil
```

Comment:
This does not make sense as you can customize the binary and detect if it's named `podman`